src/lib/DocsSearchBar.js

```diff
@@ -75,6 +75,7 @@ class DocsSearchBar {
       autoselect: true,
       ...autocompleteOptions,
     };
+    this.templates = new Templates(this.autocompleteOptions.templates || {});
```

Comment:
```suggestion
    this.templates = new Templates(this.autocompleteOptions.templates);
```

this shoulnt' be necessary when there is a default value
```javascript
function template(
options = ...
) { 
...
```